tap_aggregator/src/main.rs

```diff
@@ -55,21 +64,23 @@ async fn main() -> Result<()> {
         args.max_connections,
     )
     .await?;
+    info!("Server started. Listening on port {}.", args.port);
 
     // Have tokio wait for SIGTERM or SIGINT.
     let mut signal_sigint = signal(SignalKind::interrupt())?;
     let mut signal_sigterm = signal(SignalKind::terminate())?;
     tokio::select! {
-        _ = signal_sigint.recv() => println!("SIGINT"),
-        _ = signal_sigterm.recv() => println!("SIGTERM"),
+        _ = signal_sigint.recv() => debug!("Received SIGINT."),
+        _ = signal_sigterm.recv() => debug!("Received SIGTERM."),
     }
 
     // If we're here, we've received a signal to exit.
-    println!("Shutting down...");
+    warn!("Shutting down...");
```

Comment:
Should this be a warn? I guess it might depend what signal we get. Shutting down seems like info to me.

Reply:
Yeah wasn't sure on this one. Let's have it as info.